src/containers/Bounty/SubmissionsAndCommentsCard.module.scss

```diff
@@ -52,8 +52,16 @@
   margin: $xxl-spacing auto;
 }
 
+.borderStyle {
+  li {
+    &:after {
+      border-bottom: $base-border !important;
+      width: calc(100%) !important;
```

Comment:
Same here - there should not need to be animportant. 99.99% of times - if using an important - it's the wrong thing to do.

Reply:
these are hard coded into the table styles, so i guess i will need to create a prop to handle this